cmd/test.go

```diff
@@ -65,7 +71,16 @@ func runTest(cmd *cobra.Command, args []string) error {
 	}
 	p := project.New(pDeps, pCfg)
 
-	comp := compiler.New(logger)
+	var comp *compiler.Compiler
```

Comment:
Applying our usual style here. I scan scan code faster when it is consistent and this is also useful since we now have multiple inputs for the constructor.